example/testapp/othermodule.py

```diff
@@ -39,8 +39,8 @@ def intro():
     
     return u'はじめまして'
 
-@rpcmethod(name='view.request')
-def request(**kwargs):
+@rpcmethod(name='view.getrequest')
+def getrequest(**kwargs):
```

Comment:
Even after updating that, this threw a 500 when executed.

    Internal Server Error: /
    Traceback (most recent call last):
      File ".../rpc4django/venv3/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
        response = get_response(request)
      File ".../rpc4django/venv3/lib/python3.5/site-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
        response = self._get_response(request)
      File ".../rpc4django/venv3/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
        response = self.process_exception_by_middleware(e, request)
      File ".../rpc4django/venv3/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
        response = wrapped_callback(request, *callback_args, **callback_kwargs)
      File ".../rpc4django/venv3/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
        return view_func(*args, **kwargs)
      File ".../rpc4django/rpc4django/views.py", line 194, in serve_rpc_request
        resp = dispatcher.jsondispatch(request.body, request=request)
      File ".../rpc4django/rpc4django/rpcdispatcher.py", line 331, in jsondispatch
        return self.jsonrpcdispatcher.dispatch(raw_post_data.decode('utf-8'), **kwargs)
      File ".../rpc4django/rpc4django/jsonrpcdispatcher.py", line 128, in dispatch
        if 'request' in kwargs and inspect.getargspec(func)[0][0] == 'request':
    IndexError: list index out of range